schemas/hs00_event_histogram.fbs

```diff
@@ -0,0 +1,37 @@
+
+file_identifier "hs00";
+
+table ArrayUInt   { value: [uint];    }
+table ArrayULong  { value: [ulong];   }
+table ArrayDouble { value: [ double]; }
+
+// Allows to use multiple data format for the arrays
+union Array {
+    ArrayUInt,
+    ArrayULong,
+    ArrayDouble
+}
+
+// Meta information for one dimension
+table DimensionMetaData {
+    size: uint;        // Size of the full histogram for this dimension
+    unit: string;      // Unit for this dimension
+    label: string;     // Label for this dimension
+    bins: Array;       // Binning information for this dimension
+}
+
+// Represents the n dimensional histogram
+// Subsets of the histogram are also supported
+table EventHistogram {
+    source: string;              // Source name
+    timestamp: ulong;            // Timestamp (in ns, after unix epoch)
+    dim_metadata: [DimensionMetaData]; // Meta data for each dimension
+    timestamp_last_metadata: ulong; // Timestamp (ns, after unix epoch) when the last metadata information was written
```

Comment:
metadata_timestamp?

Reply:
or `last_metadata_timestamp`

Reply:
Yes, last should appear to be more specific!